utils.js

```diff
@@ -0,0 +1,7 @@
+const pullRequestMergeCommitRegex = /^Merge pull request #\d+ from \S+/
+const branchMergeCommitRegex = /Merge (remote-tracking )?branch \S+( of https:\/\/github\.com\/\S+)? into \S+/
+function isMergeCommitMessage (message) {
+  return !!(message.match(pullRequestMergeCommitRegex) || message.match(branchMergeCommitRegex))
```

Comment:
Adding `!!` seems like it reverse the function output? Was it just wrong before?

Reply:
It coerces the output to a boolean. Without it, we return `null` (from `.match()`) instead of `false`.

Reply:
So it was fine before, but this makes the newly-added tests more explicit.